src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/EventValidationService.java

```diff
@@ -97,6 +100,17 @@ public boolean validateCaseState(CaseDetails caseDetails) {
         return validated;
     }
 
+    public boolean validateCurrentPosition(CaseDetails caseDetails) {
```

Comment:
This approach might make the logic more readable:

private static final List<String> INVALID_STATES_FOR_CLOSED_CURRENT_POSITION = List.of(SUBMITTED_STATE, ACCEPTED_STATE, REJECTED_STATE);
...
INVALID_STATES_FOR_CLOSED_CURRENT_POSITION.contains(caseDetails.getState())